post/templates/post/post.html

```diff
@@ -33,23 +33,34 @@ <h2 class="text-center">{{ post.title }}</h2>
     <br />
     <hr>
 
-    <h4>Comments...</h4>
-    {% if not post.comments.all %}
-    No Comments Yet...<a class="btn btn-comment-custom" role="button" href="#">Add one</a>
 
-    {% else %}
-    {% for comment in post.comments.all %}
-        <strong>
-        {{ comment.name }} - 
-        {{ comment.created_at }} - {{ comment.modified_at }}
-        </strong>
+{% if post.photo %}
```

Comment:
Hey, it looks like a duplicate of a code above.

Reply:
noted